fusee/fusee-primary/src/sdmmc.c

```diff
@@ -3615,3 +3615,37 @@ bool sdmmc_card_present(struct mmc *mmc)
 {
     return mmc->card_present(mmc);
 }
+
+/**
+ * Prints out all of the tegra_mmc struct fields
+ *
+ * @mmc mmc The controller with which to dump registers.
+ */
+void sdmmc_dump_regs(struct mmc *mmc) {
+    printk("block_size: 0x%04X\n",mmc->regs->block_size);
```

Comment:
To keep the same file contents between primary and secondary, it is possible to:
1. #include <inttypes.h>
2. Change the format specifier to use the macros from inttypes.h, for example:
printk("block_size: 0x%04" PRIX16 "\n",mmc->regs->block_size);

For the 32-bit use PRIX32